presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
 
 These configuration options may require tuning in specific situations:
```

Comment:
I suggest splitting this large config options table into at least two sections: common/basic options and advanced options. The first one should contain the config options we found affecting query performance in a big way (e.g., amount of cluster/node memory , join concurrency, broadcast vs distributed, # worker threads), the other one may cover more obscure config options that are not likely to be changed a lot.